bin/arnold

```diff
@@ -320,27 +335,75 @@ function _set_k8s_env() {
         source "${customer_vars}"
     fi
 
-    # Check that the current user is already logged in k8s. Note that the
-    # `whoami` command is only compatible with openshift and should be adapted
-    # for k8s.
-    if ! ${KUBECTL} whoami &> /dev/null ; then
-        if [[ -z "${K8S_SA_TOKEN}" ]] ; then
+    # If an authentication token is provided, check that the K8S_AUTH_HOST is also provided.
+    if [[ -n "${K8S_AUTH_API_KEY}" && -z "${K8S_AUTH_HOST}" ]] ; then
+      log error "Error: you should provide K8S_AUTH_API_KEY and K8S_AUTH_HOST"
```

Comment:
The logger already prints the log type, i.e. `ERROR` so you don't need to repeat this information:

```suggestion
      log error "You should provide K8S_AUTH_API_KEY and K8S_AUTH_HOST"
```

The same suggestion applies for every logged errors.